Alignment-TUMAdsk-1/TUMAdsk-alignment-01.ifc

```diff
@@ -0,0 +1,56 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION(('ViewDefinition [Ifc4X3NotAssigned]'),'2;1');
+FILE_NAME(
+/* name */ 'TUMAdsk-alignment-01.ifc',
+/* time_stamp */ '2021-07-14T11:28:41',
+/* author */ ('Sebastian Esser'),
+/* organization */ ('Unknown'),
+/* preprocessor_version */ 'GeometryGymIFC v0.1.14.0 by Geometry Gym Pty Ltd built 2021-06-29T03:35:26',
+/* originating_system */ 'ConsoleCreateAlignment v1.0.0.0',
+/* authorization */ 'None');
+
+FILE_SCHEMA (('IFC4X3_RC4'));
+ENDSEC;
+
+DATA;
+#1= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#2= IFCAXIS2PLACEMENT3D(#1,$,$);
+#3= IFCLOCALPLACEMENT($,#2);
+#4= IFCSITE('21I4ZLo9zAqulPrGFG7kW0',$,'site',$,$,#3,$,$,$,$,$,$,$,$);
+#5= IFCPROJECT('3yPJXv7zf1huBjsiOwq7fP',$,'TUM-Adsk_alignment_01','sample for a single horizontal line segment - geometry + semantic part',$,$,$,(#20),$);
+#6= IFCRELAGGREGATES('1V4reoMpP5gPsAAN68aHfH',$,$,$,#5,(#4));
+/* Entry point for alignment container */
+#7= IFCALIGNMENT('3gB8GbZ3jAawT$yMHh8EuG',$,$,$,$,$,$,$);
+#8= IFCRELCONTAINEDINSPATIALSTRUCTURE('0okge1aQnDkA_ejxcZTeJo',$,'Site','Site Container for Elements',(#7),#4);
+/* groups all horizontal segments using an IfcRelNests rel */
+#9= IFCALIGNMENTHORIZONTAL('3aEAaXWfD0ihvEYJSKS5Nz',$,$,$,$,#29,#26,$);
+#10= IFCRELNESTS('0VR061g110I8Ym_9qcAG3S',$,$,$,#7,(#9));
+#11= IFCCARTESIANPOINT((3.0,1.0,0.0));
+/* Semantic part of the line segment */
+#12= IFCALIGNMENTHORIZONTALSEGMENT($,$,#11,0.0,0.0,0.0,56.4,$,.LINE.);
+/* link semantic segment with horizontal alignment container */
+#13= IFCALIGNMENTSEGMENT('2VYaib_VX6eRj5Pdmq6eyH',$,$,$,$,$,$,#12);
+#14= IFCRELNESTS('1_HaU0QKbAHRnjuOrkJNsm',$,$,$,#9,(#13));
+#15= IFCDIRECTION((0.0,0.0));
+#16= IFCVECTOR(#15,1.0);
+/* line geometry */
+#17= IFCLINE(#11,#16);
+#18= IFCAXIS1PLACEMENT(#11,$);
+/* Trim the entire line */
+#19= IFCCURVESEGMENT(.CONTINUOUS.,#18,IFCNONNEGATIVELENGTHMEASURE(0.0),IFCNONNEGATIVELENGTHMEASURE(56.4),#17);
```

Comment:
If the `Vector` is of length `1.0` - I'm not sure about a `NonNegativeLengthMeasure` of `56.4` ...